templates/inspec/tests/integration/build/gcp-mm.tf

```diff
@@ -494,6 +525,7 @@ resource "google_cloudfunctions_function" "function" {
   trigger_http          = "${var.cloudfunction["trigger_http"]}"
   timeout               = "${var.cloudfunction["timeout"]}"
   entry_point           = "${var.cloudfunction["entry_point"]}"
+  runtime               = "nodejs6"
```

Comment:
What's this for?

Reply:
Creating a cloud function requires a runtime to be declared now.

Reply:
Cool. In the future, this should probably be a separate PR (very unrelated to region backend services) but it's fine for now.